modules/velero-obtaining-by-accessing-binary.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="velero-obtaining-by-accessing-binary_{context}"]
+= Accessing the Velero binary in the Velero deployment in the cluster
+
+You can use a shell command to access the Velero binary in the Velero deployment in the cluster.
+
+.Prerequisites
+
+* our Data Protection Application custom resource has a status of `Reconcile complete`.
```

Comment:
Looks like you're missing a letter here. 

```suggestion
* Your Data Protection Application custom resource has a status of `Reconcile complete`.
```

Reply:
@kaovilai Capitalization differs based on usage: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
```suggestion
* Your DataProtectionApplication custom resource has a status of `Reconcile complete`.
```
Ok. then remove spacing to clarify this is k8s object reference.

Reply:
```suggestion
* our `DataProtectionApplication` custom resource has a status of `Reconcile complete`.
```

Reply:
Yes, I deleted the file you mentioned. it was empty.  Formatting changes made according to final comment.